glamod_marine_processing/qc_suite/modules/external_clim.py

```
@@ -307,13 +307,34 @@ def get_value(
         valid_indices = isvalid(lat) & isvalid(lon) & isvalid(month) & isvalid(day)
         result = np.full(lat_arr.shape, None, dtype=float)  # type: np.ndarray
 
+        if isinstance(valid_indices, (bool, np.bool)):
+            valid_indices = [valid_indices]
+
         for i in range(np.size(result)):
             if not valid_indices[i]:
                 continue
-            tindex = self.get_tindex(int(month_arr[i]), int(day_arr[i]))
+
+            mon_i = int(month_arr[i])
+            ml = get_month_lengths(2004)
+            day_i = int(day_arr[i])
+            lat_i = lat_arr[i]
+            lon_i = lon_arr[i]
+            if (
+                mon_i < 1
+                or mon_i > 12
+                or day_i < 1
+                or day_i > ml[mon_i - 1]
+                or lat_i < -180
+                or lat_i > 180
+                or lon_i < -90
+                or lon_i > 90
+            ):
+                continue
```

@jjk-code-otter: Or should we raise an error?